File: 16_Behaviour Driver Development/Praktikum/Code program soal prioritas 2/src/test/java/starter/product/Product.java
package starter.product;
import net.thucydides.core.annotations.Step;
public class Product {
    @Step("I am on the product page example pulsa")
    public void onTheProductPage(){
        System.out.println("I am on the product page example pulsa");
    }
    @Step("I enter my number")
    public void enterNumber(){
        System.out.println("I enter my number");
    }
    @Step("Product list is displayed")
    public void productList(){
        System.out.println("Product list is displayed");
    }
    @Step("I click the product i want")
    public void clickProduct(){
        System.out.println("I click the product i want");
    }
    @Step("Redirected to the payment page")
    public void redirectToPaymentPage(){
        System.out.println("Redirected to the payment page");
    }

}

File: 16_Behaviour Driver Development/Praktikum/Code program soal explorasi/cypress/e2e/Product/Product.js
import {Given , When, And, Then} from "cypress-cucumber-preprocessor/steps";

Given ("I am on the product page example pulsa",function(){
    console.log("On the product page")
});
When ("I enter my number",function(){
    console.log("Enter number")
});
And ("Product list is displayed",function(){
    console.log("List product displayed")
});
Then ("I click the product i want",function(){
    console.log("Click product")
});
And ("Redirected to the payment page",function(){
    console.log("Redirected to the payment page")
});